src/scripts/Save.ts

```diff
@@ -203,19 +199,20 @@ class Save {
         $('#saveModal').modal('hide')
     }
 
+    // TODO (@Isha) reimplement
     public static convertShinies(list: Array<string>) {
-        const converted = [];
-        for (const pokemon of list) {
-            const shiny = parseInt(pokemon['shiny']);
-            const name = pokemon['name'];
-            if (shiny == 1 && player.caughtShinyList.indexOf(name) == -1) {
-                player.caughtShinyList().push(pokemon['name']);
-                converted.push(pokemon['name']);
-            }
-        }
-        if (converted.length > 0) {
-            Notifier.notify(`You have gained the following shinies: ${converted}`, GameConstants.NotificationOption.success)
-        }
+        // let converted = [];
+        // for (let pokemon of list) {
+        //     let shiny = parseInt(pokemon['shiny']);
+        //     let name = pokemon['name'];
+        //     if (shiny == 1 && player.caughtShinyList.indexOf(name) == -1) {
+        //         player.caughtShinyList().push(pokemon['name']);
+        //         converted.push(pokemon['name']);
+        //     }
+        // }
+        // if (converted.length > 0) {
+        //     Notifier.notify("You have gained the following shinies: " + converted, GameConstants.NotificationOption.success)
+        // }
```

Comment:
Can we just comment this out as a code block?

Reply:
Yeah I will be fixing this in a big save refactor which is up next